code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -357,6 +367,17 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_desc = "A very classy looking drink."
 	shot_glass_icon_state = "shotglassred"
 
+/datum/reagent/consumable/ethanol/rock
+	name = "Rock wine"
+	description = "Pioneers are said to have drunk this. Or was it ashwalkers? dwarves? anyways..."
```

Comment:
capitalization issues in here near the end

Reply:
Also, seems a bit odd to reference ashwalkers, given this PR also kills them.

Reply:
yes thats intentional